CHANGES

```diff
@@ -1,6 +1,19 @@
 This file describes the most significant changes. For more detail, use
 'git log' on a clone of the charm repository.
 
+================================================================================
+What's new in Charm++ 6.10.1
+================================================================================
+
+This is a minor bug-fix release, with the following changes:
+
+- Fix verbs on new InfiniBand hardware/drivers. This fixes some of the observed
+  failures on InfiniBand machines, but not all of them; it is still a known
+  issue.
+
+- Avoid CmiAbort segfaults in production builds.
```

Comment:
I think this might leave the impression that CmiAbort itself was faulty, maybe formulate as "Avoid aborting with a segfault when calling CmiAbort in production builds."